doc/wg/opentitan.md

```diff
@@ -0,0 +1,49 @@
+OpenTitan Tock Working Group (OT)
+=================================
+
+- Working Group Charter
+- Adopted 2/14/2020
+
+## Goals
+
+The goals of the OpenTitan Tock Working Group (OT) are to:
+
+- complete and test RISC-V architectural support for OpenTitan,
+- design, implement and test peripheral drivers for the earlgrey platform,
+- document APIs and major design decisions, and
+- collaborate with the core group on Tock APIs that may need to change to better
+  support RISC-V/OT.
+
+## Members
+
+- Laura Abbott, Oxide Computer
+- Brad Campbell (Chair),  University of Virginia
+- Jon Flatley, Google
+- Alistair Francis, Western Digital
+- Garret Kelly, Google
+- Philip Levis, Stanford University/Google
+- Patrick Mooney, Oxide Computer
+
+## Code Purview
+
+The OT working group will be in charge of (responsible for reviewing and
+approving pull requests for) the following code directories:
```

Comment:
In light of the discussion on the call today that the OpenTitan working group should be given merge capabilities, should this be "responsible for reviewing, approving, and merging pull requests for..."

Reply:
Added some more details.